includes/node/class-webhook.php

```diff
@@ -38,6 +41,9 @@ class Webhook {
 	public static function init() {
 		add_action( 'init', [ __CLASS__, 'register_endpoint' ] );
 		add_filter( 'newspack_webhooks_request_body', [ __CLASS__, 'filter_webhook_body' ], 10, 2 );
+		if ( defined( 'WP_CLI' ) && WP_CLI ) {
+			WP_CLI::add_command( 'np-network process', [ __CLASS__, 'cli_process' ] );
```

Comment:
We use the full plugin slug as the WP CLI command base elsewhere in code. Also, the command name should be more descriptive:

 ```suggestion
				WP_CLI::add_command( 'newspack-network process-webhooks', [ __CLASS__, 'cli_process' ] );
```

Reply:
Updated with your suggestions.